_drafts/2020-07-27-data_science_plotting_notebook_template.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Jupyter Notebook Templates for Data Science: Plotting"
+description: >
+  Jupyter notebooks are great for data exploration; jumpstart your
+  visualizations with this plotting notebook!
+image: /files/jupyter-library/jupiter_red_spot_juno.jpg
+image_alt: >
+  A photo of the Library of Congress in 1902.
+categories: 
+  - jupyter
+  - my_projects
+---
+
+{% capture file_dir %}/files/jupyter-library/{% endcapture %}
+
+{% include lead_image.html %}
+
+I recently released my [Jupyter Notebook Template Library][library]. Its goal
+is to accelerate your data science projects without having to to spend hours
+poking through old notebooks to find handy code snippets. In this post I dive
+into the plotting notebook to show you what it can do.
+
+[library]: https://github.com/agude/Jupyter-Notebook-Template-Library
+
+[nb_post]: {% post_url 2016-10-17-jupyter_not_for_development %}
+
+## The Plotting Notebook
+
+Visualizing your data is a critical step in understanding your data, and so it
```

Comment:
Repeated "your data". Would say "Visualization is ..." or "a critical step in understanding it"